### ISSUE TITLE
[range.hsl] broken in plugdata

### ISSUE THREAD

**Issue by 60-hz** [timothyschoen/PlugData#1457]
The interaction doesn't work, tried from the help file.
I guess that this would be a good candidate for a pdlua graphics version too, with resize function.

**Comment by porres** [comment]
yup, can confirm that! I was gonna add this issue, but checked first if someonehad reported it

**Comment by porres** [comment]
@timothyschoen I'd like to suggest some labels for issues, it makes it really nice to organize stuff... we could have a label for 'ELSE'. If you invite me to this repository team I can help with that and organize it

**Comment by alcomposer** [comment]
Is this still an issue? I have tested side-by-side with latest plugdata & PD-Vanilla, it looks like it is functioning identical? (at least as identical for random numbers):

https://github.com/plugdata-team/plugdata/assets/12004932/33688997-f2dd-4893-83b4-142457c2e96b

**Comment by porres** [comment]
it's [range.hsl], not [range] :)

**Comment by alcomposer** [comment]
Ah ok! :-)
